src/filter-input-attributes.ts

```diff
@@ -78,9 +78,14 @@ interface IProps {
 export default function (props: IProps): {[key: string]: any} {
   const attributes: {[key: string]: any} = {};
 
-  allowedAttributes.forEach((allowedAttribute) => {
-    if (props[allowedAttribute]) {
-      attributes[allowedAttribute] = props[allowedAttribute];
+  Object.keys(props).forEach((attribute) => {
+    const isDataAttribute = attribute.startsWith('data-');
+    const isAllowedAttribute = allowedAttributes.find(
+      (allowedAttribute) => allowedAttribute === attribute
+    );
```

Comment:
```suggestion
    const isAllowedAttribute = allowedAttributes.includes(attribute);
```
Seems also be a good case for `includes`